ios/brave-ios/Sources/BraveWallet/Crypto/Stores/TransactionConfirmationStore.swift

```diff
@@ -140,6 +140,22 @@ public class TransactionConfirmationStore: ObservableObject, WalletObserverStore
         return [outputLine, valueLine, addressLines].joined(separator: "\n")
       }.joined(separator: "\n\n")
       return inputs + "\n\n" + outputs
+    } else if activeParsedTransaction.transaction.coin == .zec,
+      let zecTxData = activeParsedTransaction.transaction.txDataUnion.zecTxData
+    {
+      let inputs = zecTxData.inputs.enumerated().map { (index, input) in
+        let inputLine = "\(Strings.Wallet.inputLabel): \(index)"
+        let valueLine = "\(Strings.Wallet.valueLabel): \(input.value)"
+        let addressLines = "\(Strings.Wallet.addressLabel):\n\(input.address)"
+        return [inputLine, valueLine, addressLines].joined(separator: "\n")
+      }.joined(separator: "\n\n")
+      let outputs = zecTxData.outputs.enumerated().map { (index, output) in
+        let outputLine = "\(Strings.Wallet.outputLabel): \(index)"
+        let valueLine = "\(Strings.Wallet.valueLabel): \(output.value)"
+        let addressLines = "\(Strings.Wallet.addressLabel):\n\(output.address)"
+        return [outputLine, valueLine, addressLines].joined(separator: "\n")
+      }.joined(separator: "\n\n")
```

Comment:
Just checking if this format is intended? Differs from desktop Nightly (I'm not familiar with expectations here) cc @cypt4 

<img width="300" alt="iOS" src="https://github.com/user-attachments/assets/c4595feb-3b12-484a-b998-a1f6e4ea308e" /> | <img width="300" alt="Desktop Nightly" src="https://github.com/user-attachments/assets/73c5c4ce-b79e-4aad-bafd-64b9e905493b" />
--|--

Reply:
DMed @cypt4 he is fine with the format on iOS. we basically keep the same format as BTC send tx